tools/appimage/overlay/AppRun

```diff
@@ -0,0 +1,34 @@
+#! /bin/bash
+
+# Export APPRUN if running from an extracted image
+self="$(readlink -f -- $0)"
+here="${self%/*}"
+APPDIR="${APPDIR:-${here}}"
+
+# Export TCl/Tk
+export TCL_LIBRARY="${APPDIR}/usr/share/tcltk/tcl8.4"
+export TK_LIBRARY="${APPDIR}/usr/share/tcltk/tk8.4"
+export TKPATH="${TK_LIBRARY}"
+
+# Export SSL certificate
+export SSL_CERT_FILE="${APPDIR}/opt/_internal/certs.pem"
+
+# Get the executable name, i.e. the AppImage or the python binary if running from an
+# extracted image
+executable="${APPDIR}/opt/python3.9/bin/python3.9"
+if [[ "${ARGV0}" =~ "/" ]]; then
+    executable="$(cd $(dirname ${ARGV0}) && pwd)/$(basename ${ARGV0})"
+elif [[ "${ARGV0}" != "" ]]; then
+    executable=$(which "${ARGV0}")
+fi
+
+# This is a reimplementation of tools/start_client.sh from the base repository
+# The UI build was performed during AppImage build, so we skip that step
+(
+	set -e -x
+	export PATH="${APPDIR}/usr/bin:${PATH}"
+	cd "$APPDIR/opt/randovania"
+	"$APPDIR/usr/bin/python3" -m randovania gui --preview main
```

Comment:
They absolutely can. Since we're using the binary release from create_release.py, this line has been changed to just pass the rest of the cmdline args along. It's more or less just `/path/to/randovania "$@"` now.